src/components/WalletConnect.jsx

```diff
@@ -80,28 +80,11 @@ export default function WalletConnect() {
                     if (!wallet.alert) {
                       dispatch(setWallet({ address: displayWallet() }));
                       const owner_address = getWallet().address;
-                      let inTotalsPromise = userInTotals(owner_address);
                       let in_DB = await userInDB(owner_address);
                       if (!in_DB) {
-                        const user = await instantiateUser(owner_address);
+                        const user = instantiateUser(owner_address);
```

Comment:
I removed it because before I modified the WallectConnect.jsx for the gleam campaign, there wasn't an await there before. If I remember correctly, I had to add the await there because there was an additional firebase query for the gleam campaign. Since I removed that additional promise (firebase query), I thought I would just revert it back to how it was before (without the await). I attached a link to how I changed this specific part when I did the gleam campaign below. It is on line 70-73 in that PR.

https://github.com/Tapera-Finance/gard.money/pull/112/files#diff-7f13fc9ac48e88707b5642a771d74efa63d26258d804d6a575cfc5499b9fc0b3L70-L73